pkg/clusteragent/telemetry/collector.go

```diff
@@ -60,15 +60,17 @@ type ApmRemoteConfigEventTags struct {
 	//nolint:revive // TODO(TEL) Fix revive linter
 	RcId string `json:"rc_id"`
 	//nolint:revive // TODO(TEL) Fix revive linter
-	RcClientId string `json:"rc_client_id"`
-	RcRevision int64  `json:"rc_revision"`
-	RcVersion  uint64 `json:"rc_version"`
-	//nolint:revive // TODO(TEL) Fix revive linter
-	KubernetesClusterId string `json:"k8s_cluster_id"`
-	KubernetesCluster   string `json:"k8s_cluster"`
-	KubernetesNamespace string `json:"k8s_namespace"`
-	KubernetesKind      string `json:"k8s_kind"`
-	KubernetesName      string `json:"k8s_name"`
+	RcClientId     string             `json:"rc_client_id"`
+	RcRevision     int64              `json:"rc_revision"`
+	RcVersion      uint64             `json:"rc_version"`
+	ClusterTargets []K8sClusterTarget `json:"cluster_targets"`
```

Comment:
@liliyadd if you change the JSON structure of an APM remote config event from [what's described here in the spec](https://github.com/DataDog/instrumentation-telemetry-api-docs/blob/main/GeneratedDocumentation/ApiDocs/v2/producing-telemetry.md#example--13), we will need to update the spec and update our backend to be able to handle the new JSON fields. Do you intend these cluster targets to be saved on the backend in a new table?

Reply:
/cc @stanistan 

This is a bit related with what I mentioned in one of my comments: https://github.com/DataDog/datadog-agent/pull/25588#discussion_r1631061652